Objective-C/Internal/CBLCollationExpression.m

```diff
@@ -41,4 +41,9 @@ - (id) asJSON {
     return @[@"COLLATE", [_collation asJSON], [_operand asJSON]];
 }
 
+- (void) dealloc {
+    _collation = nil;
+    _operand = nil;
```

Comment:
I understand that it will be automatically released. Do we need to set them to nil?